README.md

```diff
@@ -435,20 +423,13 @@ If a custom annotation is required, the annotation should be documented in the p
 <a name="miscellaneous"/>
 ## Miscellaneous
 
-`and` is preferred over `&&`.
-
-`or` is preferred over `||`.
-
-`is` is preferred over `==`.
+`&&` is preferred over `and`.
 
-`not` is preferred over `!`.
+`||` is preferred over `or`.
 
-`or=` should be used when possible:
+`==` is preferred over `is`.
```

Comment:
`is` doesn't mean the same thing in ruby and i think that makes a good reason we don't use it - same with other operators